.github/workflows/ci_zkevm.yml

```diff
@@ -170,6 +170,84 @@ jobs:
           name: logs_${{ github.run_id }}
           path: ./kurtosis-cdk/ci_logs
 
+  kurtosis-cdk-type-1-pectra-pmt:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout cdk-erigon
+        uses: actions/checkout@v4
+
+      - name: Setup kurtosis
+        uses: ./.github/actions/setup-kurtosis-type1
+
+      - name: Run polycli load test
+        run: /usr/local/bin/polycli loadtest --rpc-url $(kurtosis port print cdk-v1 cdk-erigon-rpc-001 rpc) --private-key "0x12d7de8621a77640c9241b2595ba78ce443d05e94090365ab3bb5e19df82c625" --verbosity 700 --requests 500 --rate-limit 50 --concurrency 2 --mode t
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Set up envs
+        run: |
+          kurtosis files download cdk-v1 bridge-config-artifact
+          echo "BRIDGE_ADDRESS=$(/usr/local/bin/yq '.NetworkConfig.PolygonBridgeAddress' bridge-config-artifact/bridge-config.toml)" >> $GITHUB_ENV
+          echo "BridgeAddress: $BRIDGE_ADDRESS"
+          echo "ETH_RPC_URL=$(kurtosis port print cdk-v1 el-1-geth-lighthouse rpc)" >> $GITHUB_ENV
+          echo "BRIDGE_API_URL=$(kurtosis port print cdk-v1 zkevm-bridge-service-001 rpc)" >> $GITHUB_ENV
+          echo "L2_RPC_URL=$(kurtosis port print cdk-v1 cdk-erigon-rpc-001 rpc)" >> $GITHUB_ENV
+
+      - name: Clone bridge repository
+        run:  git clone --recurse-submodules -j8 https://github.com/0xPolygonHermez/zkevm-bridge-service.git -b develop  bridge
+
+      - name: Build docker image
+        run: |
+          cd bridge
+          make build-docker-e2e-real_network
+
+      - name: Run test ERC20 Bridge
+        run: |
+          #!/bin/bash
+          cd bridge
+          mkdir tmp
+          cat <<EOF > ./tmp/test.toml
+          TestL1AddrPrivate="0x12d7de8621a77640c9241b2595ba78ce443d05e94090365ab3bb5e19df82c625"
+          TestL2AddrPrivate="0x12d7de8621a77640c9241b2595ba78ce443d05e94090365ab3bb5e19df82c625"
+          [ConnectionConfig]
+          L1NodeURL="http://${ETH_RPC_URL}"
+          L2NodeURL="${L2_RPC_URL}"
+          BridgeURL="${BRIDGE_API_URL}"
+          L1BridgeAddr="${BRIDGE_ADDRESS}"
+          L2BridgeAddr="${BRIDGE_ADDRESS}"
+          EOF
+          echo "Running test with config:"
+          cat ./tmp/test.toml
+          docker run --network=host  --volume "./tmp/:/config/" --env BRIDGE_TEST_CONFIG_FILE=/config/test.toml bridge-e2e-realnetwork-erc20
+
+      - name: Upload logs
+        uses: actions/upload-artifact@v4
+        with:
+          name: evm-rpc-tests-logs-${{ github.run_id }}
+          path: ./cdk-erigon/logs/evm-rpc-tests.log
```

Comment:
Are you sure we need it? AFAIR this was using in ansible tests

Reply:
Not sure, this was in the original file, I can remove it.